### ISSUE TITLE
Calculating contingency tables without showing SE

### ISSUE THREAD

**Issue by martinctc** [gergness/srvyr#45]
Great package guys, only started exploring it but absolutely love it!

survey_mean() can be used to create proportions for multiple variables to create contingency tables. 
Is there a way of doing this without generating the SE columns as well?

Following is an example where Q10 is a multiple choice question with say e.g. 9 codes. The following generates a contingency table with rows equal to the number of "Segment"s and 19 columns. What arguments will I need in order to generate one which has 10 columns (the nine codes PLUS the grouping column)? I intend for each column to show the proportions for each code. 

main_svy_df %>%
  group_by(Segment) %>%
  summarise_at(vars(starts_with("Q10")),funs(survey_mean(.)))

Many thanks!

**Comment by gergness** [comment]
Thanks for the kind words!

Interesting point. srvyr currently isn't optimized for this because most (if not all) cases the survey design only affects variance, so you could calculate the mean with `weighted.mean()` if you don't care about SE. However, I think you're right, there should be a way to do this, I'm surprised that passing `var_type = NULL` doesn't work.

I'm not sure I totally understand your data structure, but could Q10 be re-written as a single factor variable instead of 9 dummy variables. If so, you might be able to do the following:
```r
main_svy_df %>%
  group_by(Segment, Q10_FACTOR) %>%
  summarise(survey_mean())
# And then use tidyr to spread the Q10_FACTOR levels wide
```

Below, I've played around with an example from the survey package. stype is the stand in for Segment in your pseudo-code, and the 3 api variables are for Q10*

``` r
library(srvyr)
library(dplyr)
data(api, package = "survey")

# stratified sample
dstrata <- apistrat %>%
  as_survey(strata = stype, weights = pw)


# This shouldn't give se
dstrata %>% 
  group_by(stype) %>%
  summarize_at(vars(starts_with("api")), ~survey_mean(., vartype = NULL))
#> # A tibble: 3 x 7
#>   stype api00 api00_se api99 api99_se api.stu api.stu_se
#>   <fct> <dbl>    <dbl> <dbl>    <dbl>   <dbl>      <dbl>
#> 1 E      674.     12.5  636.     13.3    355.       14.0
#> 2 H      626.     15.5  617.     15.8   1071.       83.4
#> 3 M      637.     16.6  610.     16.8    696.       50.0
```

<sup>Created on 2018-10-03 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

**Comment by martinctc** [comment]
Thanks Greg! Unfortunately it wouldn't work for my case to convert them into a single factor variable because these variables are binary multi-choice variables (i.e. not mutually exclusive). E.g.
Q10_1 = c(1,0,1,0...)
Q10_2=c(1,0,0,0....)
...
What you suggested at the end wasn't exactly what I was looking for though, as I wanted to get at weighted proportions rather than the weighted mean. I tried the following two variations, which both return se columns:

`apistrat %>%
  mutate_at("sch.wide",funs(case_when(.=="Yes"~1, TRUE~0))) %>%
  as_survey(strata = stype, weights = pw) %>%
  group_by(stype) %>%
  summarize_at(vars(sch.wide), ~survey_mean(., vartype = NULL))`

OR

`apistrat %>%
  mutate_at("sch.wide",funs(case_when(.=="Yes"~1,TRUE~0))) %>%
  as_survey(strata = stype, weights = pw) %>%
  group_by(stype) %>%
  summarise_at(vars(sch.wide),funs(survey_mean))`

So far I've used a less elegant work-around (below) to remove the SE columns, which works fine for most cases:
`%>% select(-ends_with("_se"))`

**Comment by tzoltak** [comment]
Regarding setting `var_type = NULL` doesn't work: setting argument to `NULL` results in `missing()` returning `TRUE` just as this argument hasn't been set at all.
Thus line 82 in *survey_statistics.r*:
`vartype <- if (missing(vartype)) "se" else match.arg(vartype, several.ok = TRUE)`
results in setting `vartype` to "se" in such a case.

**Comment by gergness** [comment]
Fixed in #48
